### DISCUSSION TITLE
Is it possible to use ebusd AND existing vaillant heating controller?

### DISCUSSION THREAD

**Discussion by maxromanovsky** [john30/ebusd#1021] [Q&A] [answered]
Is it possible to use ebusd AND existing vaillant heating controller?
The use case would be to be able to check status (and override config ideally) using standard Vaillant controller, but in general leave the temperature management and collect statistics using Home Assistant.

**Comment by Koky05** [comment, marked as answer]
Yes, it is possible. I use eBus and vaillant calormatic 470 controller.

**Reply by maxromanovsky** [reply]
@Koky05

1. how do you connect it: sequentially or in parallel?
2. any issues / quirks (e.g. only one controller can change values, other is read-only or something like this)?

**Reply by Koky05** [reply]
I connect it parallel and no issue related to connection. (I fight with WiFi connection but it is only mine environment problem.)

**Reply by maxromanovsky** [reply]
Thanks!
